src/app/shared/doc-viewer/doc-viewer.ts

```diff
@@ -107,12 +113,20 @@ export class DocViewer implements OnDestroy {
 
     Array.prototype.slice.call(exampleElements).forEach((element: Element) => {
```

Comment:
nit: I know this was already here, but can you update it to use the spread operator? `Array.prototype.slice.call` bothers me a little.
[go/js-style#features-arrays-spread-operator](go/js-style#features-arrays-spread-operator)